### PR TITLE
[script.module.simpleeval] 0.9.10

### PR DESCRIPTION
**Pull request by mikesilvo164**
### Description
Moving to module as requested [HERE](https://github.com/xbmc/repo-scripts/pull/1338#discussion_r396025052)
Corresponding update to Skin Shortcuts forthcoming.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.simpleeval/addon.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.simpleeval" name="Simple Eval" version="0.0.1" provider-name="SiLVO">
```
Comment (enen92) [inline script.module.simpleeval/addon.xml:2]: Versioning is wrong. Which version is this? Please use upstream versions: https://pypi.org/project/simpleeval/#history

Comment (enen92) [inline script.module.simpleeval/addon.xml:2]: author is the module author not yourself

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml:2]: I have no idea which version this is. How do I find that out?

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml:2]: @enen92 I did not create this script, I am just maintaining it. I do not know which version of simpleeval is being used ATM.

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml:2]: Will change this in next push

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml:2]: Hmmm... I think I am just gonna update the code and use the latest version

Comment (enen92) [reply script.module.simpleeval/addon.xml:2]: yeah that's the way to go. Update to latest upstream version and check if your script needs any adaption :)

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml]: Everything looks good after testing so I think I'm good 🤞

File: script.module.simpleeval/addon.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+    <extension point="xbmc.python.module" library="lib" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Simple Eval</summary>
+        <description lang="en_GB">A library for easily adding evaluatable expressions into python projects</description>
+        <platform>all</platform>
+        <license>MIT</license>
+        <source>https://github.com/mikesilvo164/script.module.simpleeval</source>
```
Comment (enen92) [inline script.module.simpleeval/addon.xml:12]: an icon would be nice, this is still shown in the interface. You can use one of the default/placeholders:

https://github.com/xbmc/repo-scripts/blob/matrix/script.module.bottle/icon.png

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml:12]: Ok I will add one now and force push

Comment (mikesilvo164) [reply script.module.simpleeval/addon.xml:12]: @enen92 Done 👍

**Pull request merged**